lessons/Data-Science/dot-plots/langs/en-us/index.adoc

```diff
@@ -0,0 +1,273 @@
+}
+
+
+=== Investigate
+
+
+@lesson-instruction{
+- Turn to @printable-exercise{interpret-dot-plots.adoc} and complete the first section: Reading a Dot Plot.
+- Be prepared to discuss your answers with the class.
+}
+
+@teacher{Review students' responses as a class. Questions 1, 2, and 3 touch on three relevant concepts: range, mode, and proportional reasoning. }
+
+Now that we are comfortable reading dot plots, we need a common vocabulary to discuss the data that they display. To describe the distribution of data--the way that it is spread out on a number line--it is helpful to locate any outliers, clusters, peaks, and gaps.
+
+- A *cluster* is a group of data points that are close together.
+- A *gap* is an interval where there are no data points.
+- An *outlier* occurs when one data point is much larger or smaller than the other data points.
+- A *peak* is the value(s) with the most data.
+
+@lesson-instruction{
+
+@center{@image{images/names1.png, 250}}
+
+- Let's complete the second section of @printable-exercise{interpret-dot-plots.adoc} together using the data in the dot plot for Group A.
+}
+@QandA{
+@Q{What peaks should we label?}
+@A{There is a peak at 4.}
+
+@Q{What clusters should we label?}
+@A{There is a cluster from 3 to 6.}
+
+@Q{What gaps should we label?}
+@A{There are gaps at 7 and 10.}
+
+@Q{What outliers should we label?}
+@A{There is an outlier at 12.}
+
+@Q{Now let's turn to question 5. What do those peaks, clusters, gaps, and outliers tell us about the dataset?}
+}
+
+@lesson-instruction{
+Complete the third section with your partner.
+}
+
+@teacher{
+Discuss and review students' responses. Students will complete the final section of @printable-exercise{interpret-dot-plots.adoc} after a brief class discussion on typicality.
+}
+
+Another way of describing data on a dot plot is to think about its typicality.
+
+@QandA{
+@Q{Let's think about the word "typical". Describe a "typical" morning for you.}
+@A{Invite students' to share. Emphasize that "typical" is "the usual", or "what's expected", but it is not always a perfect predictor. It may be "typical" to eat breakfast at 7am, but there are probably days where you eat a little bit earlier or a little bit later - or even much earlier or much later!}
+
+@Q{What does the word "typical" mean to you?}
+}
+
+@lesson-instruction{
+Complete the final section of @printable-exercise{interpret-dot-plots.adoc}.
+}
+
+@teacher{Review students' responses, emphasizing that there are multiple ways to decide what is typical in a dataset. You may want to highlight a few different and appropriate responses to highlight that we are simply _estimating_ typicality.}
```

Comment:
@retabak Now that you have the close focusing on a shared dataset, please provide some more scaffolding here to support teachers.